projects/packages/jetpack-mu-wpcom/src/features/wpcom-dashboard-widgets/wpcom-daily-writing-prompt/index.js

```diff
@@ -0,0 +1,82 @@
+import apiFetch from '@wordpress/api-fetch';
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import moment from 'moment';
+
+import './style.scss';
+
+export default () => {
+	const [ prompts, setPrompts ] = useState( [] );
+	const [ index, setIndex ] = useState( 0 );
+	useEffect( () => {
+		// See projects/packages/jetpack-mu-wpcom/src/features/wpcom-block-editor-nux/src/blogging-prompts-modal/index.js
+		const path = addQueryArgs( `/wpcom/v3/blogging-prompts`, {
```

Comment:
I only tested simple so far, does this work for atomic ...

Reply:
Also a bit weird that there's no data layer or something, random apiFetch calls scatters like that are hard to evolve and reason about.

Reply:
I agree, let's polish this once we have most dashboard functionality in.

Reply:
> I only tested simple so far, does this work for atomic ...

Yes, it works on Atomic :)